CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
+## 1.1.1
```

Comment:
I suppose that you should use `1.1.0`. 1.1.0 is declared in pbiviz.json and package.json